.styluaignore

```diff
@@ -1,9 +1,12 @@
-/build
+/build/
+/.deps/
 /runtime/lua/coxpcall.lua
 /runtime/lua/vim/_meta
 /runtime/lua/vim/re.lua
 
-/test/functional
+test/functional/ui/decorations_spec.lua
+test/functional/ui/float_spec.lua
+test/functional/ui/multigrid_spec.lua
```

Comment:
Alternatively, we could also consider using `-- stylua: ignore start` at the beginning of each file rather than listing files to exclude here in `.styluaignore`, which might make it a bit easier for autoformatters (stylua) to correctly recognize files to ignore.

Reply:
> which might make it a bit easier for autoformatters (stylua) to correctly recognize files to ignore

Eh, I don't think we need to take this into account. Let's just consider this a minor bug that will probably be fixed soon.

Assuming both methods work equally well: I don't mind either, so if anyone has a strong preference for either one I'm fine to go with that. I guess a nice thing with ignoring with `.styluaignore` is that all the ignored files are neatly in one single place. Another nice thing is that we don't need to modify vendored files such as `/runtime/lua/vim/re.lua`. So I think I'd lean towards keeping the files in `.styluaignore`.

Tbh this is not really a big issue. If stylua happens for format `build` and whatnot isn't *that* big of a problem. It's mostly about reducing the maintenance of cmake files but it's so minor that I don't think we need to put in more time into this IMO.

Reply:
> If stylua happens for format `build` and whatnot isn't _that_ big of a problem

`lintlua` complaining about `.deps/*` can be quite noisy.

Reply:
Wut? lintlua does not lint `.deps`.